microsoft-edge/webview2/concepts/platform-components.md

```diff
@@ -0,0 +1,322 @@
+## Prerelease SDK with preview browser channel, or Release SDK with Runtime
+
+| Version | Renderer platform | Description |
+|:---|:---|:---|
+| Prerelease SDK | A preview channel of Microsoft Edge (Beta, Dev, or Canary) | For experimenting and testing your app against upcoming changes, on your Dev machines. |
+| Release SDK | The WebView2 Runtime | For shipping your app to end users. |
+
+* A Prerelease version of the WebView2 SDK uses a preview channel of Microsoft Edge (Beta, Dev, or Canary).
+* A Release version of the WebView2 SDK uses the WebView2 Runtime.
+
+
+See also:
+* [Understanding the options at the Runtime download page](./distribution.md#understanding-the-options-at-the-runtime-download-page) in _Distribute your app and the WebView2 Runtime_.
+* [Prerelease and release SDKs for WebView2](./versioning.md)
+* [Distribute your app and the WebView2 Runtime](./distribution.md)
+
+
+<!-- ====================================================================== -->
+## Using a Prerelease SDK and experimental APIs with a Preview channel of Microsoft Edge
+
+To develop the prerelease version of your app using experimental APIs, or to test your app against upcoming SDK changes:
+
+* On your Dev machine, in the Visual Studio project, install a **Prerelease** version of the `Microsoft.Web.WebView2` SDK NuGet package.  Write code that uses the **experimental** APIs (and stable APIs).
+* On your Dev machine, install and use a preview channel of Microsoft Edge.
+
+To distribute your prerelease app to your test machine:
+* On your test machine, install a preview channel of Microsoft Edge.
+
+See also:
+* [Understand the different WebView2 SDK versions](./versioning.md) - Either use a prerelease SDK with a preview channel of Microsoft Edge, or use a release SDK with the WebView2 Runtime.
+
+
+<!-- ====================================================================== -->
+## Using a Release SDK and stable APIs with the Runtime
+
+To develop the release version of your app:
+* On your Dev machine, in the Visual Studio project, install a **Release** version of the `Microsoft.Web.WebView2` SDK NuGet package.  Write code that uses only the **stable** APIs.
+* On your Dev machine, use the WebView2 Runtime (part of the SDK package).
+
+The WebView2 Runtime is like a browser engine for use as a component in your app.
+
+There are several ways to distribute your app and the Runtime to users.  See [Ways to distribute, install, and update the Runtime on the user's machine](#ways-to-distribute-install-and-update-the-runtime-on-the-users-machine) above.
+
+
+See also:
+* [Understand the different WebView2 SDK versions](./versioning.md) - Either use a prerelease SDK with a preview channel of Microsoft Edge, or use a release SDK with the WebView2 Runtime.
+
+
+<!-- ====================================================================== -->
+## How the WebView2 SDK is laid out in relation to how the WebView2 Runtime is laid out
+
+todo: what's the intention here?
```

Comment:
what's the intention of "How the WebView2 SDK is laid out in relation to how the WebView2 Runtime is laid out"?